### DISCUSSION TITLE
After install the last version of firefly 2.0.1 ASMB y SMR dissappear

### DISCUSSION THREAD

**Discussion by highchall** [iotaledger/firefly#7549] [Help & Support]
After install the last version of firefly 2.0.1  in MAC OS Ventura  ASMB y SMR dissappear.
In the wallet I just can see IOTA.

Any suggestiuon about how to fix it?

**Comment by MiKeF05** [comment]
No need to fix this. To claim your SMR token you need the Firefly Shimmer wallet. You can download it here https://firefly.iota.org/. ASMB will be converted into IOTA tokens.

**Comment by begonaalvarezd** [comment]
Quick information about ASMB and SMR:

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.2 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but they were not the real shimmers. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

Will close the issue because the new firefly app is not intended to display old staking rewards, but the new iota airdrop is on the new app and the shimmer tokens should be managed from the shimmer app. 

Please let us know if you have any other issues
